src/algorithms/digi/SiliconTrackerDigi.h

```diff
@@ -47,14 +46,7 @@ namespace eicrecon {
     std::shared_ptr<spdlog::logger> m_log;
 
     /** Random number generation*/
-    TRandomMixMax m_random;
-    std::function<double()> m_gauss;
-
-    // FIXME replace with standard random engine
-    //std::default_random_engine generator; // TODO: need something more appropriate here
-    //std::normal_distribution<double> m_normDist; // defaults to mean=0, sigma=1
-
-    //algorithms::Generator m_rng = algorithms::RandomSvc::instance().generator();
+    algorithms::Generator m_rng = algorithms::RandomSvc::instance().generator();
```

Comment:
In the factory you will have to make sure this is available. It will get auto-generated even if it isn't available, but we want EICrecon's JANA2 interfaces to have all services instantiated in advance. See e.g. https://github.com/eic/EICrecon/blob/14bc49be8ad313ab2311fbbc08163d2cf81dfab7/src/factories/calorimetry/CalorimeterHitDigi_factory.h#L35

This then initializes the random service a single time at https://github.com/eic/EICrecon/blob/14bc49be8ad313ab2311fbbc08163d2cf81dfab7/src/services/algorithms_init/AlgorithmsInit_service.h#L64

Reply:
Thanks!